infra/cifuzz/stack_parser_test.py

```diff
@@ -32,26 +32,31 @@ class ParseOutputTest(unittest.TestCase):
 
   def test_parse_valid_output(self):
     """Checks that the parse fuzzer output can correctly parse output."""
-    test_output_path = os.path.join(TEST_FILES_PATH,
-                                    'example_crash_fuzzer_output.txt')
-    test_summary_path = os.path.join(TEST_FILES_PATH, 'bug_summary_example.txt')
+    # Read the fuzzer output from disk.
+    fuzzer_output_path = os.path.join(TEST_FILES_PATH,
```

Comment:
nit: you want to create a utils file that reads and return content of files, i see this content reading happen in a couple of places.

Reply:
It's challenging to do this in a file because the TEST_FILES_PATH is different in every test file.
The only part that can really be shared is joining a dir and reading a file. Not really that worthwhile.